tests/zfs-tests/tests/functional/cache/cache_012_pos.ksh

```diff
@@ -94,7 +94,7 @@ typeset write_max2=$(get_tunable L2ARC_WRITE_MAX)
 
 log_must test $write_max2 -eq $write_max
 
-log_must set_tunable32 L2ARC_WRITE_MAX $(( 64 * 1024 * 1024 ))
+log_must set_tunable32 L2ARC_WRITE_MAX $(( 256 * 1024 * 1024 ))
 export RUNTIME=1
 
 typeset do_once=true
```

Comment:
@gamanakis could you support me here on test motivation? Looks like we hope that after random reads some data from l2arc will be discarded, and it's only a probability. Interesting thing is that some CI runners won't usually get to this condition with relatime=on, my local tests don't reproduce it. So, looks like read test is related to actual write on TXG? I've tried to maximize probability though, no luck in CI.

Reply:
That's interesting, this test clearly does depend on some write IO to force the L2 ARC discards, and it seems those writes are for the atime updates.  @gmelikov for the purposes of this test case you could enable atime updates by default and add a small comment.